### DISCUSSION TITLE
zx dosen't work on windows10 terminal

### DISCUSSION THREAD

**Discussion by PARADISELIN** [google/zx#322] [Help needed]
## Expected Behavior
works fine in window terminal

## Actual Behavior
It works fine in git bash, but in windows ternimal it gives error, I don't know what I need to set in windows terminal.

## Steps to Reproduce the Problem
![bug](https://user-images.githubusercontent.com/17696492/156087854-4abf9677-5cc7-4918-9213-fb08b6f64e42.png)

## Specifications

- Version: 5.2.0
- Platform: windows10

**Comment by powerfulyang** [comment]
[$.shell](https://github.com/google/zx#:~:text=Configuration-,%24.shell,-Specifies%20what%20shell)
https://stackoverflow.com/questions/68931619/how-to-use-google-zx-to-execute-ffmpeg-shell-in-windows#:~:text=How%20to%20use%20google/zx%20to%20execute%20ffmpeg%20shell%20in%20windows%3F

**Reply by powerfulyang** [reply]
看下文档哦，亲
